### ISSUE TITLE
Feature request: Add cron arguments into logging

### ISSUE THREAD

**Issue by jordie23** [wp-cli/cron-command#41]
Would be great if the `Executed the cron event 'x'` message included a dump of the arguments passed in. Perhaps as part of a debug group so can be enabled/disabled via the `--debug` flag?

i.e. this line
https://github.com/wp-cli/cron-command/blob/master/src/Cron_Event_Command.php#L262

Personal reasoning is it'd be good to see the post ID for any `publish_future_post` events, but I'm sure it'd come in handy for others wanting to debug.

**Comment by schlessera** [comment]
Yes, sounds like a good change, using debug output for backwards compatibility.

Are you interested in providing a PR for this?
